option.go

```diff
@@ -37,6 +37,14 @@ func OptionPrefix(x string) Option {
 	}
 }
 
+// OptionInitialText to set the initial buffer text
+func OptionInitialText(x string) Option {
```

Comment:
I feel `OptionInitialBufferText` is clear and precise. What do you think?

Reply:
yes, makes sense. i have renamed the function.